esi_leap/api/controllers/v1/utils.py

```diff
@@ -42,6 +42,13 @@ def check_resource_lease_admin(cdict, resource, project_id,
                 if start_time >= parent_lease.start_time and \
                    end_time <= parent_lease.end_time:
                     return parent_lease_uuid
+                else:
```

Comment:
Not relevant to this PR, do we need another exception message for sub-sub-leases?

Reply:
Ah, nope! Sub-sublessees are not allowed.